libs/constants/src/enums/set-qualifier.enum.ts

```diff
@@ -0,0 +1,11 @@
+export enum SetQualifier {
+  INCLUDE = 0,
+  EXCLUDE = 1,
+  AT_LEAST_ONE = 2
+}
+
+export const SetQualifierName: ReadonlyMap<SetQualifier, string> = new Map([
+  [SetQualifier.INCLUDE, 'Include'],
```

Comment:
Is there every a reason for "Include" here? Feel like this could just be an boolean where true is "Required"/"At least one" and false is "Exclude"